app/assets/stylesheets/application.scss

```diff
@@ -15,6 +15,7 @@
 @import 'material';
 @import 'rails_bootstrap_forms';
 @import 'badge';
+@import 'organisations';
```

Comment:
this is the line that imports organisations.scss, which is empty
how about we only create these and import these if there's content in these?